src/timeline/widgets/VideoItem.cc

```diff
@@ -71,26 +72,10 @@ VideoItem::VideoItem(QSharedPointer<MatrixClient> client,
                      QWidget *parent)
   : QWidget(parent)
   , url_{url}
-  , text_{QFileInfo(filename).fileName()}
+  , text_{filename}
   , client_{client}
 {
-        Q_UNUSED(data);
-        readableFileSize_ = calculateFileSize(QFileInfo(filename).size());
+        readableFileSize_ = utils::humanReadableFileSize(data->size());
```

Comment:
`QIODevice::size` is not guaranteed to be the total size of the data.

Reply:
Fixed in commit f159b7b.